safe_commands.py

```
@@ -1,89 +1,85 @@
-            'success': False
-        }
+        return {"stdout": "", "stderr": str(e), "returncode": -1, "success": False}
+
 
 def safe_git_add():
     """Safely add all changes"""
-    return run_command(['git', 'add', '.'])
+    return run_command(["git", "add", "."])
+
 
 def safe_git_commit(message: str, details: List[str] = None):
```

Ruff (0.15.6)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)